posts/templates/posts/_post.html

```diff
@@ -37,23 +39,35 @@
     <div class="actions">
         {% if request.user.is_authenticated %}
             <a class="mdi-action-grade{% if post.extra.is_favorited %} favorite{% endif %}"
-               href="{% url 'posts:favorite' pk=post.original_or_self.pk %}"></a>
+               href="{% url 'posts:favorite' pk=post.original_or_self.pk %}">
+            </a>
             {% if post.extra.can_be_reposted %}
-                <a class="mdi-av-repeat"
-                   href="{% url 'posts:repost' pk=post.original_or_self.pk %}"></a>
+                <a class="mdi-av-repeat" href="{% url 'posts:repost' pk=post.original_or_self.pk %}"></a>
             {% endif %}
+            <a class="mdi-content-reply" href="#reply-area-{{ post.pk }}" data-toggle="collapse"></a>
             <div class="pull-right">
                 {% if post.extra.can_be_edited %}
-                    <a class="mdi-editor-mode-edit"
-                       href="{% url 'posts:edit' pk=post.original_or_self.pk %}"></a>
+                    <a class="mdi-editor-mode-edit" href="{% url 'posts:edit' pk=post.original_or_self.pk %}"></a>
                 {% endif %}
                 {% if post.extra.can_be_deleted %}
-                    <a class="mdi-action-delete"
-                       href="{% url 'posts:delete' pk=post.original_or_self.pk %}"></a>
+                    <a class="mdi-action-delete" href="{% url 'posts:delete' pk=post.original_or_self.pk %}"></a>
                 {% endif %}
             </div>
             <div class="clearfix"></div>
         {% endif %}
     </div>
+    <div class="collapse{% if display_collapse %} in{% endif %}" id="reply-area-{{ post.pk }}">
+        <hr/>
+        {% include 'posts/_reply_post.html' %}
```

Comment:
I would only include the `_reply_post.html` in `post_detail.html` since I don't want to view the replies anywhere else, right? This could make it simpler to just include `_post.html` to render a view, instead of creating the html once more like in `_replied_posts.html`.